### PR TITLE
regina keku

### PR DESCRIPTION
**Pull request by henceagrin36**
added a good LOTR quote 👍

### CODE REVIEW

File: favorite_movie_quotes.txt [hunk 1]
```diff
@@ -6,3 +6,4 @@ You’ve got a friend in me.
 まだまだだね.
 Welcome to Fight Club. The first rule of Fight Club is: you do not talk about Fight Club.
```
Comment (rohjay) [inline favorite_movie_quotes.txt:8]: *nitpick* ~ There's some odd whitespace here...

**Review by rohjay — CHANGES_REQUESTED** [review]
One small change and we're good =]

**Comment by rohjay** [comment]
Heyyyy Regina! You cleanin' up for anything special? 😉

I hope you're well and life is good =]

**Comment by henceagrin36** [comment]
hey Ryan, sorry just saw this reply! hope you're doing well too, and that you're the same awesome human being (if not more)!

no real reason for closing this haha. thanks for teaching me about git back in the day – and look! now I can close PRs too!

**Comment by rohjay** [comment]
Awwww you're gonna make me blush! It was my pleasure - it's great to see you out rockin' in your career. I know it's not my place, and my hand had very little to do with it, but I'm proud of you! You always were a smart cookie =]

Keep on keepin' on, Regina. Reach out if you need anything from an old mentor!

**Pull request closed without merging**